api/include/opentelemetry/context/context.h

```diff
@@ -0,0 +1,169 @@
+#pragma once
+
+#include <atomic>
+
+#include "opentelemetry/context/context_value.h"
+#include "opentelemetry/nostd/shared_ptr.h"
+#include "opentelemetry/nostd/string_view.h"
+#include "opentelemetry/trace/key_value_iterable_view.h"
```

Comment:
Some include nits: I think `atomic` and `key_value_iterable_view.h` are not needed.

Reply:
I have removed them.